websites/M/MoodlePresence/dist/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": " .•` KIΛRΛ `•. ",
+        "id": "586905336850677760"
+    },
+    "contributors": [
+        {
+            "name": "ΛΚΘRΛ",
+            "id": "387655649934311427"
+        }
+    ],
+    "service": "Moodle",
+    "version": "1.0.0",
+    "color": "#ffbc4f",
+    "description": {
+        "en": "Your end-to-end online learning platform for K12, higher education and workplace training. The most used open source LMS in the world."
+    },
+    "tags": [
+        "moodle",
+        "education",
+        "media"
+    ],
+    "category": "socials",
+    "url": "moodle.com",
+    "logo": "https://i.imgur.com/ltzuG6b.png",
+    "thumbnail": "https://i.imgur.com/ltzuG6b.png",
+    "regExp": "moodle.(.*?)[.](.*?)",
```

Comment:
Okay, problem with this regExp is, not every moodle instance is on moodle subdomain. Sometimes moodle is on main domain or even in folder like ``example.com/moodle``. My school have moodle in this way: ``example.com/moodle`` - will not work for me. So i don't really know how you want detect this thing to work everywhere.